src/robot/model/body.py

```diff
@@ -254,7 +275,8 @@ def flatten(self) -> 'list[BodyItem]':
         return steps
 
 
-class Body(BaseBody):
+class Body(BaseBody["Keyword", "For", "While", "If", "Try", "Return", "Continue",
+                    "Break", "Message", "Error"]):
```

Comment:
I'm totally nitpicking, but we've generally used single quotes around strings when possible.